apache2.yaml

```diff
@@ -143,9 +155,6 @@ subpackages:
       - runs: |
           set -x
 
-          # Install the entrypoint script
-          install -Dm755 entrypoint.sh "${{targets.subpkgdir}}"/usr/bin/entrypoint.sh
-
```

Comment:
why is entrypoint script being removed?

if this is unintentional, it should be added back in.

Reply:
The file httpd-foreground is what we referred to as entrypoint.sh. In the upstream image, there is no reference to entrypoint but httpd-foreground which is why it was removed. The content is however thesame for entrypoint.sh and httpd-foreground